src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 0;
+  padding: 0 8px;
+}
+
+.label {
+  position: relative;
+  display: block;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
```

Comment:
Do not hard code height - use padding instead

Reply:
Thanks, done